### DISCUSSION TITLE
raise limit on land and water from 320ft

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17739] [Feature requests] [answered]
![Github Park 2022-08-06 06-19-46](https://user-images.githubusercontent.com/94884086/183244821-1188fce2-1157-4d17-8f79-d361ce5203d8.png)
[Github Land.zip](https://github.com/OpenRCT2/OpenRCT2/files/9274823/Github.Land.zip)

currently land and water are restricted to a maximum of 320 ft, I propose we allow raising land and water close to the 590 ft height limit of the game

**Comment by astraylife1** [comment]
Yes please

**Comment by 733737** [comment, marked as answer]
this was changed by https://github.com/OpenRCT2/OpenRCT2/pull/19988
